qiime2/sdk/usage.py

```diff
@@ -243,10 +585,27 @@ def init_data_collection(self, ref, collection_type, *records):
         value = self._init_data_collection_(ref, collection_type, *records)
         return self._push_record(ref, value, 'init_data_collection')
 
-    def _init_data_collection_(self, ref, collection_type, *records):
+    def _init_data_collection_(self, ref, collection_type, records):
         raise NotImplementedError
 
-    def merge_metadata(self, ref, *records):
+    def merge_metadata(self, ref: str,
+                       *records: typing.List['ScopeRecord']) -> 'ScopeRecord':
+        """
+        Create a `ScopeRecord` for merged Metadata.
```

Comment:
Technically all of these plugin-dev-facing methods create `ScopeRecord`s. I prefer the language you have adopted above, omitting the mention of the `ScopeRecord`.